site/model_help.md

```diff
@@ -0,0 +1,99 @@
+## The Model Help Tool
+
+The Model Help Tool provides information about the folders and attributes that are valid for sections and folders of a domain model. This is useful when creating a new domain model, or expanding an existing model, including discovered models.
+
+**NOTE:** The Model Help Tool was added as part of WebLogic Deploy Tooling 1.8.
+
+Here is a simple example using the Model Help Tool:
+```yaml
+<wls-deploy-home>/bin/modelHelp.sh -oracle_home /tmp/oracle -path resources:/JDBCSystemResource
+```
+This will result in the following output:
+```yaml
+Path: resources:/JDBCSystemResource
+
+  Valid Attributes:
+    CompatibilityName           string
+    DeploymentOrder             integer
+    DeploymentPrincipalName     string
+    DescriptorFileName          string
+    ModuleType                  string
+    Notes                       string
+    SourcePath                  string
+    Target                      delimited_string
+
+  Valid Folders:
+    JdbcResource
+    SubDeployment
+```
+This result lists the attributes and folders available for the `JDBCSystemResource` folder in the `resources` section of the model. This information could be used to construct this model section:
+```yaml
+resources:
+    JDBCSystemResource:
+        CompatibilityName: 'myName'
+        DeploymentOrder: 5
+        Target: 'ms1,ms2'
+        JdbcSystemResource:
+            # JdbcSystemResource attributes and folders
+        SubDeployment:
+            # SubDeployment attributes and folders
+```
+
+### Path Patterns
+There are a number of ways to specify model location in the `-path` argument. Here are some examples:
+ 
+List all the top-level model sections, such as `topology`, `resources`, etc.:
+```yaml
+-path top
+```
+ 
+List the attributes and folders within a section, such as `topology`, `resources`, etc.:
+```yaml
+-path topology
+```
+ 
+List all the attributes and folders within a folder:
+```yaml
+-path resources:/JDBCSystemResource/JdbcResource
+```
+ 
+If the section is not provided for a folder, then it will be derived and included in the output text:
+```yaml
+-path /JDBCSystemResource/JdbcResource
+```
+
+### Output Options
+There are several command-line options that can be used to control the output text for the model path. Only one of these options can be used at a time. If no output options are specified, then the attributes and immediate sub-folders for the specified path are listed.
```

Comment:
options that can be used to control -> options that you can use to control
Only one of these options can be used at a time -> Use only one of these options at a time